system/diagnostic_graph_utils/src/node/logging.cpp

```diff
@@ -33,8 +33,13 @@ LoggingNode::LoggingNode(const rclcpp::NodeOptions & options) : Node("logging",
   sub_graph_.register_create_callback(std::bind(&LoggingNode::on_create, this, _1));
   sub_graph_.subscribe(*this, 1);
 
+  pub_error_graph_ =
+    create_publisher<tier4_debug_msgs::msg::StringStamped>("~/debug/error_graph", rclcpp::QoS(1));
```

Comment:
@isamu-takagi 
I temporarily used error_graph. Is there any official or better name?

Reply:
Nothing in particular, but since it is a string type, something like `error_graph_text` might be better.

Reply:
Fixed here
https://github.com/autowarefoundation/autoware.universe/pull/9421/commits/618244cd569275cbd17c6e2097a8f6f422db8f0a